<ui version="4.0">
 <class>MainMenu</class>
 <widget class="QWidget" name="MainMenu">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QPushButton" name="hostButton">
     <property name="text">
      <string>Host Game</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="joinButton">
     <property name="text">
      <string>Join Game</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="singlePlayerButton">
     <property name="text">
      <string>Single player</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="optionsButton">
     <property name="text">
      <string>Options</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="statsButton">
     <property name="text">
      <string>Statistics/Highscore</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="exitButton">
     <property name="text">
      <string>Exit</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>exitButton</sender>
   <signal>clicked()</signal>
   <receiver>MainMenu</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>329</x>
     <y>370</y>
    </hint>
    <hint type="destinationlabel">
     <x>500</x>
     <y>387</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
